tests/datasets/test_patternnet.py

```diff
@@ -6,6 +6,7 @@
 from pathlib import Path
```

Comment:
Is the new `PatternNet.zip` file intentional? If the MD5 has changed we should update it in `tests/datasets/test_patternnet.py`. The MD5 isn't actually tested because we override `download_url` and don't use the MD5, but I might fix that someday.

Reply:
the md5 has not changed I believe. When I run the code with download=True, it now works smoothly, whereas before, you would get no such file error after the download.

Reply:
I mean the MD5 of the fake data file used for testing, not the actual MD5 for the real data.

Reply:
That one is already correct.

Reply:
But the file changed, so therefore the MD5 must have changed too?

Reply:
I checked it again, the current MD5 is correct for the fake data file. I am not sure what I did wrong then.

Reply:
On `main`, the MD5 of this file is `5649754c78219a2c19074ff93666cc61`. On `patternet`, it is `43f80a17fd07f3eb6059069c8565de90`.